doc/03-monitoring-basics.md

```diff
@@ -664,6 +664,14 @@ The following macros provide global statistics:
   icinga.num\_hosts\_in\_downtime     | Current number of hosts in downtime.
   icinga.num\_hosts\_acknowledged     | Current number of acknowledged host problems.
 
+### Access Environment Variables via Runtime Macros <a id="env-runtime-macros"></a>
+
+All environment variables of the Icinga process are available as runtime macros
+named `env.<env var name>`. E.g. `$env.PATH$` for $PATH. Especially useful on
+Windows where `%ProgramFiles%` doesn't get resolved by Icinga, but
+`$env.ProgramFiles$` does. In contrast to the above runtime macros just
+`$ProgramFiles$` doesn't work. Env vars require the "env." prefix.
```

Comment:
My idea was to not mention `%ProgramFiles%` because I think it provides no extra value here and might look like an example to a sloppy reader.

On the other hand, requiring the `env.` prefix is relevant information, unless there's another place in the documentation that would make this clear (like a list what scopes are searched if the macro isn't fully qualified, but I didn't find something like that).

Reply:
It **is** an example. A good example. Don't you agree?

Reply:
It's a negative example. There's already a positive example with a clearly Windows-specific variable `$env.ProgramFiles$`, so what would be the benefit of showing another syntax that doesn't work in this context?